_config.yml

```diff
@@ -1,16 +1,18 @@
-title: "Jekyll Plugins • Community-built plugins to level up your Jekyll experience."
+title: "Jekyll directory • Community-built themes and plugins to level up your Jekyll experience."
 repository: https://github.com/jekyll/plugins
 url: http://jekyll.github.io
-baseurl: /plugins
+baseurl: /directory
```

Comment:
This seems to have broken everything.

Do we need to rename the repository at the same time we make this change?